{"id": "5340", "title": "Bug 5340: Cancelling add exception breakpoint has no effect", "description": "", "OB": "When adding an exception breakpoint in the debug perspective, cancelling the process has no effect. The progress dialog closes immediately, and eventually, the exception list comes up, ignoring the cancellation request.", "EB": "The progress dialog should honor the cancellation request when adding an exception breakpoint. It should either cancel the process entirely or disable the cancel button. To achieve this, when calling ProgressMonitorDialog.run, the 'cancellable' parameter should be set to false.", "SR": "To reproduce: 1) Go to the breakpoints pane in the debug perspective. 2) Click the 'Add Exception' button. 3) In the progress dialog, click the 'Cancel' button."}